modules/navier_stokes/test/tests/finite_volume/ins/mixing_length_eddy_viscosity_aux/mixing_length_eddy_viscosity.i

```diff
@@ -3,7 +3,7 @@ von_karman_const = 0.41
 H = 1 #halfwidth of the channel
 L = 150
 
-Re = 13700
+Re = 100
```

Comment:
I don't get this, shouldn't mixing length be used to push our Re through the laminar limit and still be stable?

Reply:
Marking this conversation as resolved as we can discuss this on the original thread @GiudGiud opened